test/functional/tests/lazy_writes/cleaning_policy/test_cleaning_params.py

```diff
@@ -0,0 +1,205 @@
+#
+# Copyright(c) 2019-2021 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+#
+
+import time
+
+import pytest
+
+from api.cas import casadm
+from api.cas.cache_config import (
+    CacheMode,
+    CleaningPolicy,
+    FlushParametersAcp,
+    SeqCutOffPolicy,
+    FlushParametersAlru
+)
+from core.test_run import TestRun
+from storage_devices.disk import DiskType, DiskTypeSet, DiskTypeLowerThan
+from test_tools.dd import Dd
+from test_utils.os_utils import Udev
+from test_utils.size import Size, Unit
+from test_utils.time import Time
+
+
+@pytest.mark.require_disk("cache", DiskTypeSet([DiskType.optane, DiskType.nand]))
+@pytest.mark.require_disk("core", DiskTypeLowerThan("cache"))
+def test_cleaning_policy_config():
+    """
+        title: Test for setting cleaning policy parameters.
+        description: |
+          Verify that cleaning policy parameters are set correctly
+          and perform simple cleaning verification.
+        pass_criteria:
+          - Parameters displayed in CLI are reflecting the values set.
+          - No data is flushed when using NOP policy.
+          - Flushing occurs in expected time range for policies that flush data.
+          - Set amount of data is flushed in a single iteration.
+    """
+    time_idle = Time(seconds=30)
+    time_wake_up = Time(seconds=10)
+    time_buffer = Time(seconds=2)
+    data_to_flush = Size(50, Unit.Blocks4096)
+    data_to_flush.set_unit(Unit.Byte)
+
+    wait_before_flush = time_idle - time_buffer  # no flushes before that
+    wait_after_flush = time_idle + time_wake_up  # flushes certain
+
+    with TestRun.step("Prepare partitions."):
+        cache_dev = TestRun.disks["cache"]
+        cache_dev.create_partitions([Size(100, Unit.MiB)])
+        core_dev = TestRun.disks["core"]
+        core_dev.create_partitions([Size(200, Unit.MiB)])
+        Udev.disable()
+
+    with TestRun.step("Start cache and add core."):
+        cache = casadm.start_cache(cache_dev.partitions[0], force=True, cache_mode=CacheMode.WB)
+        cache.set_seq_cutoff_policy(SeqCutOffPolicy.never)
+        core = cache.add_core(core_dev.partitions[0])
+
+    with TestRun.step("Change cleaning policy to NOP."):
+        cache.set_cleaning_policy(CleaningPolicy.nop)
+        if cache.get_cleaning_policy() != CleaningPolicy.nop:
+            TestRun.fail("NOP cleaning policy is not set!")
+
+    with TestRun.step("Fill core with dirty data."):
+        dd = (
+            Dd().input("/dev/zero")
+                .output(core.path)
+                .block_size(Size(1, Unit.MiB))
+                .oflag("direct")
+        )
+        dd.run()
+
+    with TestRun.step("Check core statistics."):
+        core_dirty_before = core.get_dirty_blocks()
```

Comment:
This step is actually not checking anything, it just fetches some data. Maybe merge it with the next one, which in fact checks this data against another data?